models/balancer/arbitrum/balancer_v2_arbitrum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,51 @@
+{{
+    config(
+        schema = 'balancer_v2_arbitrum',
+        alias='pools_tokens_weights',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["metacrypto", "jacektrocinski"]\') }}'
+    ) 
+}}
+
```

Comment:
Since the model is defined in `dbt_project.yml`, we can remove the `schema` here, and add `unique_key` and `materialized`, etc, for incremental strategy.
```suggestion
{{
    config(
        alias='pools_tokens_weights',
        materialized = 'incremental',
        file_format = 'delta',
        incremental_strategy = 'merge',
        unique_key = ['pool_id', 'token_address'],
        post_hook='{{ expose_spells(\'["arbitrum"]\',
                                    "project",
                                    "balancer",
                                    \'["metacrypto", "jacektrocinski"]\') }}'
    )
}}

```

Reply:
The schema is needed here since the v2 schema is not defined `dbt_project.yml`.